docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -98,6 +98,188 @@ Interventions
+  {P(\text{VAD prevalence} \mid \text{fortification})}
+  \approx \frac{1}{0.45\: (0.19, 1.05)}
+  \approx 2.22\: (0.95, 5.26).
+
+.. note::
+
+	We are modeling the reciprocal of the relative risk reported in the 
+	Cochrane review.
+
+	Additionally, this effect size crosses the null, and therefore, in some 
+	draws it will cause increasing coverage of the intervention to *increase* 
+	vitamin A deficiency prevalence. This is a limitation caused by the low 
+	quality evidence regarding the relative risk of vitamin A fortification on 
+	vitamin A prevalence. However, on average, increasing coverage of vitamin 
+	A fortification will decrease VAD.
+
+To model the uncertainty in this estimate, the above RR should be drawn from a
+`lognormal <https://en.wikipedia.org/wiki/Log-normal_distribution>`_
+distribution with median = 2.22, 2.5\ :superscript:`th`-percentile = 0.95, and
+97.5\ :superscript:`th`-percentile = 5.26. This distbibution can be created
+using `SciPy's lognorm function
+<https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.lognorm.html>`_
+as follows:
+
+.. code-block:: Python
+
+  from numpy import log
+  from scipy.stats import norm, lognorm
+
+  # median and 0.975-quantile of lognormal distribution for RR
+  median = 2.22
+  q_975 = 5.26
+
+  # 0.975-quantile of standard normal distribution (=1.96, approximately)
+  q_975_stdnorm = norm().ppf(0.975)
+
+  mu = log(median) # mean of normal distribution for log(RR)
+  sigma = (log(q_975) - mu) / q_975_stdnorm # std dev of normal distribution for log(RR)
+
+  # Frozen lognormal distribution for RR, representing uncertainty in our effect size
+  # (s is the shape parameter)
+  rr_distribution = lognorm(s=sigma, scale=median)
+
+.. note::
+
+	I copied this from Nathaniel's documentation for the folic acid RR, but I 
+	think that the same approach is appropriate. Perhaps we can eventually 
+	create a separate page that lists similar strategies that we can reference 
+	via links.
+
+Additionally, as described in the research considerations above, the 
+intervention effect is dependent on age and time since intervention coverage.
+
+For simulants covered by *baseline coverage*, the effect of the intervention 
+is applied continuously from the start of the simulation to the end of the 
+simulation and only depends on age, such that:
+
+.. code-block:: Python
+
+  if age_i < 0.5:
+  	rr_i = 1 + (rr - 1) * a_i / 0.5
+  else:
+  	rr_i = rr
+
+For simulants covered by the *intervention scale-up*, the effect of the 
+intervention will scale up linearly from zero at the start of coverage to the 
+full effect after six months of coverage and does *not* depend on age (
+research assumption), such that:
+
+.. code-block:: Python
+
+  if coverage_time_i < 0.5:
+  	rr_i = 1 + (rr - 1) * coverage_time_i / 0.5
+  else:
+  	rr_i = rr
+
+.. note::
+
+	Any ideas from the research team on how we might want to combine
+	the impact of age_i < 0.5 and coverage_time_i < 0.5 here?
```

Comment:
Hmm, I'll have to think about this some more. I was assuming we'd so something like: If we have reached p_1% of the age scale-up and p_2% of the coverage time scale-up, then the overall effect will be p_1*p_2% of the total effect (the "percents" should really be proportions, but it's easier to say percent.).

Reply:
Yeah, I was thinking this too, but then we get some weird cases on the margins
. I will draw it out and share it in the LSFF slack channel.